front/src/modules/project/utils.ts

```diff
@@ -9,10 +11,10 @@ const checkProjectFields = (
   funders: boolean;
   budget: boolean;
 } => ({
-  name: !project.name || project.name.length > 128,
-  objectives: (project.objectives ?? '').length > 4096,
-  description: (project.description ?? '').length > 1024,
-  funders: (project.funders ?? '').length > 255,
+  name: checkNameInvalidity(project.name),
+  objectives: checkFieldInvalidity(4096, project.objectives),
```

Comment:
Avoid using magic numbers. You can set for instance (in utils/strings.ts for instance):
```
export const TEXT_AREA_MAX_LENGTH = 4096;
export const TEXT_INPUT_MAX_LENGTH = 255;
export const SMALL_INPUT_MAX_LENGTH = 128;
```
and then:
```suggestion
  objectives: checkFieldInvalidity(TEXT_AREA_MAX_LENGTH, project.objectives),
```

Reply:
Done